diff -Naur xfce4-terminal-0.6.1/terminal/terminal-gdbus.c xfce4-terminal-0.6.1.tpg/terminal/terminal-gdbus.c
--- xfce4-terminal-0.6.1/terminal/terminal-gdbus.c	2013-01-01 12:09:30.000000000 +0000
+++ xfce4-terminal-0.6.1.tpg/terminal/terminal-gdbus.c	2013-03-13 19:06:28.483434248 +0000
@@ -89,7 +89,7 @@
         {
           g_dbus_method_invocation_return_error (invocation,
               TERMINAL_ERROR, TERMINAL_ERROR_OPTIONS,
-              error->message);
+              error->message, "%s");
           g_error_free (error);
         }
       else
